### PR TITLE
Add Russian and Ukrainian as active languages 

### PR DESCRIPTION
**Pull request by Darkboy17**
This PR closes/references issue #14.
## How has this been tested?

 I have fixed the issue as requested by adding the two languages, "Russian" and "Ukranian" to the Active language list.

Please take a look at the screenshot of the newly added languages.

activelanguages

To ensure if the languages can be translated, please have a look at the screenshot where I have selected "Russian" as underlined.

lang_sel_russian

Now to confirm, when I visit the link http://127.0.0.1:8000/orga/event/, it takes me to a page as shown in the screenshot below, which successfully translated the language to Russian.

page_russian_translated

Now, let us test for the language "Ukranian" by similarly following the above steps. 
Selecting "Ukranian" as shown in the screenshot below

lang_sel_ukr

Similarly, when I visit http://127.0.0.1:8000/orga/event/, I get the page in Ukranian as shown in the screenshot below.

page_ukr_translated

## Checklist

- [x] I have manually tested the functionality to cover my changes.
- [x] I have strictly followed the contribution guidelines and best practices.

### CODE REVIEW

**Comment by mariobehling** [comment]
Please test if the languages show up as an option for forms and provide a screenshot.

Screenshot from 2024-02-18 04-13-20

**Comment by Darkboy17** [comment]
Okay, I will do that and revert back soon.

**Comment by Darkboy17** [comment]
> Please test if the languages show up as an option for forms and provide a screenshot.
> 
> Screenshot from 2024-02-18 04-13-20

Here's a screenshot from my machine to confirm that the languages do show up as options in the form.

languages_in_forms

**Comment by Darkboy17** [comment]
> Please test if the languages show up as an option for forms and provide a screenshot.
> 
> Screenshot from 2024-02-18 04-13-20

Here's a screenshot from my machine to confirm that the languages do show up as options in the form.

**Review by mariobehling — APPROVED** [review]

File: src/pretalx/settings.py [hunk 1]
```diff
@@ -376,68 +377,94 @@ def merge_csp(*options, config=None):
...
         "name": _("Simplified Chinese"),
         "natural_name": "简体中文",
         "official": False,
-        "percentage": 94,
+        "percentage": 90,
         "public_code": "zh",
     },
+    "ru": {
+        "name": _("Russian"),
+        "natural_name": "Russian",
```
Comment (norbusan) [inline src/pretalx/settings.py:456]: That is not the natural name, it should be русский язык and similar for Ukraine.

File: src/pretalx/settings.py [hunk 2]
```diff
@@ -376,68 +377,94 @@ def merge_csp(*options, config=None):
...
+        "official": True,
+        "percentage": 100,
```
Comment (norbusan) [inline src/pretalx/settings.py:458]: If it 100% translated? I highly doubt

**Review by norbusan — CHANGES_REQUESTED** [review]

**Comment by norbusan** [comment]
Also, in surprised that they're are commits of me in the branch, but I guess GitHub can figure that out.

**Comment by Darkboy17** [comment]
I have made the changes as requested. Please check from your end and revert.

File: src/pretalx/settings.py (hunk 2 above)
Comment (Darkboy17) [reply src/pretalx/settings.py]: I have made the changes as requested. Please check from your end and revert back.

File: src/pretalx/settings.py (hunk 1 above)
Comment (Darkboy17) [reply src/pretalx/settings.py]: I have made the necessary changes as requested. Please check and revert back.

**Review by norbusan — CHANGES_REQUESTED** [review]
Can you please make a pull request based on the current development branch that does NOT contain all the unrelated changes?
90% of the changes in this PR are unrelated, and I am loss reading all this

**Comment by Darkboy17** [comment]
> Can you please make a pull request based on the current development branch that does NOT contain all the unrelated changes? 90% of the changes in this PR are unrelated, and I am loss reading all this

Your talking about this issue here, right?

**Comment by norbusan** [comment]
There are so many issues, here, I really ask you to start over again and be a bit more careful:
- the last two commits are for a different issue (#17)
- the last two issues fail becasue tinymce is not installed, please check the above CI errors yourself!
- there are unrelated changes that I don't know why they are there, at least in the github output

I suggest:
- clean up your git branchs, branch from current development, and add only the relevant commits from this branch (use `git cherry-pick`)
- create two new PRs from the respective new branches (copy the explanation from this branch to the new one, since it had all the screenshots etc, just click edit and copy the text)
- after the new PRs are opened, close this one with a comment referencing the new ones.

Thanks

**Comment by Darkboy17** [comment]
Hello Norbusan. I have started over again by creating two new PRs. In total, I have worked on three issues till now, one has been verified and merged.

Please review the two new PRs as referenced below :

issue #49 and issue #50.


Thank You for your suggestions.

**Comment by Darkboy17** [comment]
Hello, Norbusan. Most of the tests that failed were due to tinymce library not being found. This happened because, the newly added library which exists in my local machine specifically in the project folder does not exist in the fork I created on github. I need to add this library to my fork.

How do yo think I should approach this because the PR #49 requires much more work than the other PRs I have created.

I'm new to this, so please guide and advise.

**Comment by Darkboy17** [comment]
Hello, Norbusan. After countless hours of figuring out using your advice, by cherry-picking commits and then pushing them to their relevant branches, the past commits keep appearing on my new branches, preventing me from creating new PRs.

Please revert asap as this is hindering me from moving forward.

Please

**Pull request closed without merging**